docs/reference/commandline/service_create.md

```diff
@@ -21,46 +21,46 @@ Usage:  docker service create [OPTIONS] IMAGE [COMMAND] [ARG...]
-      --stop-grace-period value          Time to wait before force killing a container (default none)
-  -t, --tty                              Allocate a pseudo-TTY
-      --update-delay duration            Delay between updates
-      --update-failure-action string     Action on update failure (pause|continue) (default "pause")
-      --update-max-failure-ratio value   Failure rate to tolerate during an update
-      --update-monitor duration          Duration after each task update to monitor for failure (default 0s)
-      --update-parallelism uint          Maximum number of tasks updated simultaneously (0 to update all at once) (default 1)
-  -u, --user string                      Username or UID (format: <name|uid>[:<group|gid>])
-      --with-registry-auth               Send registry authentication details to Swarm agents
-  -w, --workdir string                   Working directory inside the container
+      --constraint stringSlice             Placement constraints
+      --container-label list               Container labels (default [])
+      --endpoint-mode string               Endpoint mode (vip or dnsrr)
+  -e, --env list                           Set environment variables (default [])
+      --env-file list                      Read in a file of environment variables (default [])
+      --group stringSlice                  Set one or more supplementary user groups for the container
+      --health-cmd string                  Command to run to check health
+      --health-interval duration           Time between running the check (default none)
+      --health-retries int                 Consecutive failures needed to report unhealthy
+      --health-timeout duration            Maximum time to allow one check to run (default none)
+      --help                               Print usage
+      --hostname string                    Container hostname
+  -l, --label list                         Service labels (default [])
+      --limit-cpu NanoCPUs                 Limit CPUs (default 0.000)
+      --limit-memory MemoryBytes           Limit Memory (default 0 B)
+      --log-driver string                  Logging driver for service
+      --log-opt list                       Logging driver options (default [])
+      --mode string                        Service mode (replicated or global) (default "replicated")
+      --mount mount                        Attach a filesystem mount to the service
+      --name string                        Service name
+      --network stringSlice                Network attachments
+      --no-healthcheck                     Disable any container-specified HEALTHCHECK
+  -p, --publish list                       Publish a port as a node port (default [])
+      --replicas uint                      Number of tasks (default none)
+      --reserve-cpu NanoCPUs               Reserve CPUs (default 0.000)
+      --reserve-memory MemoryBytes         Reserve Memory (default 0 B)
```

Comment:
Discussing with @dnephin;

We can use

- `bytes` for `MemoryBytes`
- `list` for `stringSlice`
- `decimal` for `NanoCPUs`

I think those are the important ones (`mount` may be ok to keep, because it is quite a detailed format)